build.gradle

```diff
@@ -11,24 +11,24 @@ buildscript {
     // changing them. Please note that, for using in Android-Components, the
     // versions below must match the ones in that repository.
     ext.versions = [
-        android_gradle_plugin: '7.4.1',
+        android_gradle_plugin: '7.4.2',
         coroutines: '1.6.4',
         jna: '5.13.0',
         junit: '4.12',
         mockito: '3.11.2',
         mockwebserver: '4.9.1', // This is different than a-c, but we're fine, it's only tests.
-        kotlin: '1.8.20',
-        robolectric: '4.9.2',
+        kotlin: '1.8.21',
```

Comment:
@rvandermeulen we recently were reminded that changing these values does have impact on consumers outside of Android Component consumers (e.g. VPN et al). Nothing to do here, but I thought sharing this concern with you as well :)

Reply:
Interesting, that's definitely a new wrinkle!

Reply:
To summarize the Slack discussion we had about this, I'll add a changelog entry for this (and future Kotlin bumps).